ArduSub/Parameters.h

```diff
@@ -361,10 +391,10 @@ class ParametersG2 {
     // proximity (aka object avoidance) library
     AP_Proximity proximity;
 #endif
-
+#if AP_SUB_RC_ENABLED
     // RC input channels
     RC_Channels_Sub rc_channels;
-
+#endif
```

Comment:
Need to keep rc_channels to avoid segfaults.

Reply:
fixed